lib/liquid/partial_cache.rb

```diff
@@ -0,0 +1,41 @@
+module Liquid
+  class PartialCache
+    def self.for(*args)
```

Comment:
Might seem useless, but I think this helps imply that caches are per-context. Eg. `PartialCache.for(context...)`

Reply:
Rather than a class, what do you think about a singleton `#load`, like

```
def self.load(template_name, context:, parse_context:)
```

Saves a few allocs, and I don't really see much value in the class instance here.